Fakes/Fakes/Networking.generated.swift

```diff
@@ -261,6 +261,40 @@ extension JetpackUser {
         )
     }
 }
+extension JustInTimeMessage {
+    /// Returns a "ready to use" type filled with fake values.
+    ///
+    public static func fake() -> JustInTimeMessage {
+        .init(
+            siteID: .fake(),
+            messageID: .fake(),
+            featureClass: .fake(),
+            ttl: .fake(),
+            content: .fake(),
+            cta: .fake()
```

Comment:
Not a blocker, for these `ttl` or `cta` namings, I think it would be clearer if we use the full name instead of these abbreviations. Without the necessary context, we cannot know what it means at once (I had to look for ttl myself 😅 )

Reply:
@toupper Yeah I know what you mean... I don't really like the abbreviations, they come from the API.

There is some value for understandability and findability, if we use the same names as the API does. It's certainly helped me understand the existing implementations, searching Calypso and Jetpack repos for cta and ttl.

`ttl` will probably be removed before the end of the project: we don't actually need it for the dismissals, it turns out.

`cta`... I don't really like the nested structure of this, or `content`, within the model. I think that within the Networking layer it's simplest to follow the API definition, but I'm going to experiment with translating it to a more view-friendly model in the Yosemite layer, rather than typealiasing the Networking model.

So for now, I'll just merge this, if that's OK with you. I'll also spell out the abbreviations in the doc comments in my next PR.